JPS_VIRTUALSENSOR/AermodAgent/src/main/java/com/cmclinnovations/aermod/AermodAgent.java

```diff
@@ -166,7 +173,10 @@ public void processRequestParameters(DerivationInputs derivationInputs, Derivati
         }
 
         aermod.createAermodInputFile(scope, nx, ny, srid);
-        aermod.createAERMODReceptorInput(scope, nx, ny, srid);
+        // The receptor.dat file may have been previously created by running AERMAP. If
+        // so, it should not be overwritten.
+        if (Files.notExists(simulationDirectory.resolve("aermod/receptor.dat")))
```

Comment:
this is minor, but in general I'd avoid hardcoding slashes because they are OS specific

Reply:
I changed that line to avoid using the slash.